shared/src/business/utilities/DateHandler.ts

```diff
@@ -781,3 +781,7 @@ export const roundDateDownToNearestHour = (isoDateString: string) => {
   formattedDate.setMilliseconds(0);
   return formattedDate;
 };
+
+export const getCurrentDateTimeInMillis = (): number => {
+  return DateTime.now().toMillis();
+};
```

Comment:
We use `Number(formatNow(FORMATS.UNIX_TIMESTAMP_MS))` to do this elsewhere in the app.

```suggestion
export const getCurrentDateTimeInMillis = (): number => {
  return Number(formatNow(FORMATS.UNIX_TIMESTAMP_MS));
};

```